packages/blocks/src/page-block/edgeless/connector-manager.ts

```diff
@@ -969,7 +969,7 @@ export class EdgelessConnectorManager {
       const start = this._getConnectorEndElement(connector, 'source');
       const end = this._getConnectorEndElement(connector, 'target');
 
-      const [startPoint, endPoint] = this._computeStartEndPoint(connector);
+      const [startPoint, endPoint] = this._computeStartEndpoint(connector);
```

Comment:
Note that "endpoint" itself is a non-synthetic word, which can be confused with the `endPoint` here (within the context of `startPoint` / `endPoint`). Could you try to avoid the `startPoint` / `endPoint` naming to 
reduce the potential cost of understanding?

Reply:
Sure, I'll try to avoid it.